make.py

```diff
@@ -9,14 +9,22 @@
 
 # Startup
 
-exec(open(os.path.expanduser('~/.emscripten'), 'r').read())
 stage_counter = 0
 
+def which(program):
+  for path in os.environ["PATH"].split(os.pathsep):
+    exe_file = os.path.join(path, program)
+    if os.path.exists(exe_file):
+      return exe_file
+  return None
+
 def build():
+  EMSCRIPTEN_ROOT = os.environ.get('EMSCRIPTEN_ROOT')
```

Comment:
I see that `emmake` will set `EMSCRIPTEN`. So perhaps we should look for that here, or maybe we should set `EMSCRIPTEN_ROOT` there, so that `emmake make.py` would work?

Reply:
Sounds good, that would be consistent with BINARYEN_ROOT

Reply:
Did we add `EMSCRIPTEN_ROOT` to emscripten so that this works?

Reply:
Agreed, yeah. Sorry, I wasn't clear earlier - would running `emmake` now set that env var, so that ammo can find it? Or do we still need to add it to there?